### PR TITLE
Fixes improper use of from_envvar and logging setup

### PR DESCRIPTION
**Pull request by dividehex**

### CODE REVIEW

File: taskclustercoalesce/web.py [hunk 1]
```diff
@@ -18,27 +18,39 @@
...
-        REDIS_URL="redis://localhost:6379",
-        PREFIX="coalesce.v1.",
-        DEBUG=False))
+def setup_logging(app):
+    stream_handler = logging.StreamHandler(sys.stdout)
+    formatter = logging.Formatter('[%(asctime)s] [%(process)d] ' +
+                                  '[%(levelname)s] %(message)s',
+                                  datefmt='%Y-%m-%d %H:%M:%S +0000')
+    stream_handler.setFormatter(formatter)
+    lvl = logging.DEBUG if os.getenv('DEBUG') is 'True' else logging.INFO
```
Comment (djmitche) [inline taskclustercoalesce/web.py]: Probably want `if os.getenv('DEBUG') else ..` here, to match `gunicorn.py`

File: taskclustercoalesce/web.py [hunk 2]
```diff
@@ -18,27 +18,39 @@
...
+    # Load setting from config.py
     # Valid environment types are Production, Testing, Development (Default)
     environment_type = os.getenv('ENVIRONMENT_TYPE', 'Development')
     app.logger.info('ENVIRONMENT_TYPE set to {0}'.format(environment_type))
     app.config.from_object('config.config.{0}'.format(environment_type))
 
-    # Override with environment vars if they exist
-    app.config.from_envvar('REDIS_URL', silent=True)
-    app.config.from_envvar('PREFIX', silent=True)
-    app.config.from_envvar('DEBUG', silent=True)
```
Comment (djmitche) [inline taskclustercoalesce/web.py]: Why didn't this work?

**Comment by djmitche** [comment]
Looks good

File: taskclustercoalesce/web.py (hunk 2 above)
Comment (dividehex) [inline taskclustercoalesce/web.py]: config.from_envvar is used to import a path to the config file, not individual vars themselves.

Comment (djmitche) [inline taskclustercoalesce/web.py]: Hah, oops.  OK :)

**Pull request merged**